src/coreclr/src/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>ilc</AssemblyName>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
```

Comment:
Are we going to keep packaging the compiler as self-contained app? I think we should. Otherwise, people would need to have matching shared preview runtime to use it.

If we keep packaging the compiler as self-contained app, I think it would be best to build and publish against latest stable released runtime (e.g. netcoreapp3.1 at this point) to avoid dealing with potential instability of current build.

Reply:
I would prefer that too but I'll have to see if we have an SDK that we can use for building. The infra people insisted that we move crossgen2 off NetCore3.1.

Self-hosting the compiler might also be an interesting option give the data I gathered on crossgen2 perf, although it has the same potential instability issues.